src/hooks/v2/contractReader/V1ProjectIdOf.ts

```diff
@@ -9,7 +9,7 @@ import { readProvider } from 'constants/readProvider'
 import { readNetwork } from 'constants/networks'
 import { ContractReadResult } from './V2ContractReader'
 
-export function useV1ProjectOf(
+export function useV1ProjectIdOf(
```

Comment:
there might be an even clearer name we could use here. something like `useV1projectIdOfV2project`? at first glance i was wondering if this returned the same argument the hook accepts